sklearn/feature_extraction/dict_vectorizer.py

```diff
@@ -105,10 +130,18 @@ def fit(self, X, y=None):
             objects) to feature values (strings or convertible to dtype).
         y : (ignored)
 
+        Raises
```

Comment:
Thanks :) Copied it from existing sources, e.g. http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.RANSACRegressor.html#sklearn.linear_model.RANSACRegressor.fit (defined here: https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/linear_model/ransac.py#L214)